lib/constants.js

```diff
@@ -14,6 +14,8 @@ const constants = {
     LOG_DBG: 5,
     DEFAULT_PORT: 8126,
     DEFAULT_WORKERS: 0,
+    DEFAULT_CACHE_MODULE: 'cache_fs',
+    DEFAULT_CACHE_PATH: '.unity_cache',
```

Comment:
`DEFAULT_CACHE_PATH` is the only real new thing here, and technically would only be used in a situation where you specified a config file that was not based on the built in template (i.e., a blank file) because there is also a per-module `cache_path` that can be defined. Anyhow, I'm inclined not to make the situation more confusing and leave it out of the docs for now.